hedyorg/hedy
# [UI] Enables multi-selecting debug lines by pressing shift key

**Description**
In this PR we make a small QoL improvement by adding an option to multi-select lines for debugging when pressing the shift key. When the shift key is pressed when clicking the line number all lines between the nearest (above) selected debug line and the current one are set as debug lines. Removing the need for selecting all these lines manually.

**Fixes**
This PR fixes #3644.

**How to test**
Verify that you can multi-select debug lines by keeping the shift key pressed when clicking the line number. De-selecting multiple lines is not possible.
